utils/triggerFormValidationErrorMessages.ts

```diff
@@ -0,0 +1,9 @@
+export const triggerFormValidationErrorMessages = (className: string) => {
+    //We are going to find all the inputs from the form to blur them to show errors for submitted doctors for the admin users
+    const modInputFieldElements = Array.from(document.body.getElementsByClassName(className))
+    //This will foucs and then blur each input so that we can see the error messages on validation and in this case the semicolon is necessary for the function to run correctly and have no errors
```

Comment:
```suggestion
    //This will focus and then blur each input so that we can see the error messages on validation and in this case the semicolon is necessary for the function to run correctly and have no errors
```